app/src/main/java/org/mifos/mobilebanking/ui/fragments/BeneficiaryApplicationFragment.java

```diff
@@ -320,6 +321,8 @@ public void showError(String msg ) {
     @Override
     public void showProgress() {
         showProgressBar();
+        llApplicationBeneficiary.setVisibility(View.GONE);
```

Comment:
This is not a good place to add this line. This method should only do what it is named.

Reply:
@miPlodder I see what you are recommending, I should do the manipulation within the presenter, not the fragment.